using-mobile-extensions/adobe-target/README.md

```diff
@@ -261,54 +261,61 @@ Target.prefetchContent(prefetchMboxesList, profileParameters, prefetchStatusCall
 {% tab title="iOS" %}
 #### Objective C
 
-Use `prefetchContent` to send a prefetch request to your configured Target server with the `ACPTargetPrefetchObject` array and specified `profileParameters`. The callback will be invoked when the prefetch is complete, which returns a success status for the prefetch request.
-
-//TODO Make changes for prefetch
+Use `prefetchContent` to send a prefetch request to your configured Target server with the `ACPTargetPrefetchObject` array and specified `ACPTargetParameters`. The callback will be invoked when the prefetch is complete, which returns null if prefetch completed sucessfully or error message for the prefetch request.
```

Comment:
Please reword the 2nd sentence so that it reads as follows:

"The callback is invoked when the prefetch is complete and, if successful, returns a null value. If the prefetch is not successful, an error message is displayed."

Reply:
Thanks @rekrishn , i will make these changes on PR#81 where this is merged as a commit.